query-engine/js-connectors/js/js-connector-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jkomyno/prisma-js-connector-utils",
-  "version": "0.0.1",
```

Comment:
The npm version was previously incremented locally with no publicly available commits. In fact, the previous published version was `0.0.7`